store/rootmulti/store.go

```diff
@@ -1068,7 +1068,9 @@ func (rs *Store) GetCommitInfo(ver int64) (*types.CommitInfo, error) {
 func (rs *Store) flushMetadata(db dbm.DB, version int64, cInfo *types.CommitInfo) {
 	rs.logger.Debug("flushing metadata", "height", version)
 	batch := db.NewBatch()
-	defer batch.Close()
+	defer func() {
+		_ = batch.Close()
```

Comment:
why have this instead of only batch.Close()?

Reply:
because otherwise the error isn't checked. (here too, but now it's explicitly not checked)